app/views/analytics/index.html.erb

```diff
@@ -153,11 +153,45 @@
     </div>
   </div><!-- row   -->
 
-  <!-- Comments List-->
   <div class="row">
-    <div class="col-sm-12">
+  <!-- Editors List-->
+    <div class="col-sm-4">
       <div class="chart-wrapper">
         <div class="chart-title background-orange">
+          Most Recent Editors - Last 30 days <%= image_tag('help_icon.png', data: {:toggle => 'popover', :container => 'body', :trigger => 'hover',:placement => "bottom", :title => "Active Editors", :content => "This metric shows the number of cases added or updated by editors in the past 30 days."})%>
+        </div>
+        <div class="chart-stage">
+         <table class='table leader-list'>
+           <tr>
+            <th></th>
+            <th>Editor</th>
+            <th>Updates</th>
+           </tr>
+
+           <% @updates.group(:whodunnit).order('count_id DESC').limit(10).count(:id).each_with_index do |who, i| %>
+            <tr>
+                <% unless who[0].to_i == 0 %>
```

Comment:
Please add a comment explaining why this is necessary.  I don't understand why who is an array, and I can't tell if that's due to our app or the ticker gem.

Reply:
`@updates.group(:whodunnit).order('count_id DESC').limit(10).count(:id)` returns an array that includes the user_id and the number of updates belonging to he/she. so, who[0] returns the user_id and who[1]returns the number of updates that user is responsible for.

The unless allows us to return an "anonymous" in the instance that no user_id is present, eg when the change is made via the console or the rails_admin panel.

Let me know if there is a better way to do these 2 things.

Reply:
Ok, that makes sense.  You might want to leave a comment in the code explaining that.

Also...we probably shouldn't be making changes via the console.  That's going to mess with our data later, particularly in terms of determining usage.

Reply:
This line would make Demeter sad. An analytics_helper would be the first place I'd think of putting these queries, and put that into a method named `ten_most_recent_editors`.

Reply:
In that case @campgurus, we'd create some type of account like `washpo-database-import` so that we could track those cases still in case of a problem.